tests/test_generators/input/kitchen_sink.yaml

```diff
@@ -240,6 +244,17 @@ slots:
   city:
   has birth event:
     range: BirthEvent
+  deprecated:
```

Comment:
I'm not sure if I filled all these fields out correctly, or if they're even useful. Maybe it doesn't really matter in this case.